htmlbook/inline_quoted.html.erb

```diff
@@ -8,9 +8,15 @@ when :superscript %><sup<%= attr?('role') ? %( #{attr 'role'}) : nil %>><%= @tex
 when :subscript %><sub<%= attr?('role') ? %( #{attr 'role'}) : nil %>><%= @text %></sub><%
 when :double %><%= class_attr ? %(<span#{class_attr}>&#8220;#{@text}&#8221;</span>) : %(&#8220;#{@text}&#8221;) %><%
 when :single %><%= class_attr ? %(<span#{class_attr}>&#8216;#{@text}&#8217;</span>) : %(&#8216;#{@text}&#8217;) %><%
-when :asciimath, :latexmath
-  open, close = ::Asciidoctor::INLINE_MATH_DELIMITERS[@type] %><%= %(<span data-type="tex">#{@text}</span>) %><%
-else %><% 
+when :latexmath
+  open, close = ::Asciidoctor::INLINE_MATH_DELIMITERS[@type] %><% if (@text.start_with? '\(') && (@text.end_with? '\)') %><%= %(<span data-type="tex">#{@text}</span>) %>
```

Comment:
Nitpick: might be a little easier to read if the `<% if` started on a new line

Reply:
Agreed. I updated the commit to do this. Note that the opening `<%` has to stay on the same line or a newline is added in the output.